samples/model-builder/test_constants.py

```diff
@@ -197,3 +197,9 @@
 )
 PYTHON_MODULE_NAME = "trainer.task"
 MODEL_TYPE = "CLOUD"
+
+# Feature store constants
+FEAUTURESTORE_NAME = "featurestore_sample"
```

Comment:
For class init, the arg `featurestore_name` can take both fully qualified resource name, as well as a resource ID. For create method, the `featurestore_id` takes only ID, not the fully qualified resource name. 

Prefer to have two constants, `FEAUTURESTORE_NAME` and `FEAUTURESTORE_ID` and use them in corresponding settings.

Reply:
yep, sounds good